hack/go.mod.default

```diff
@@ -1,24 +1,34 @@
 module github.com/vdaas/vald
 
-go 1.14
+go 1.15
 
 replace (
+	cloud.google.com/go => cloud.google.com/go master
 	github.com/Azure/go-autorest => github.com/Azure/go-autorest v14.2.0
-	github.com/aws/aws-sdk-go => github.com/aws/aws-sdk-go v1.33.21
+	github.com/aws/aws-sdk-go => github.com/aws/aws-sdk-go master
```

Comment:
aws-sdk-go is downgraded because master points v1.25.x, so i think it is better to write v1.33.xx here.